lib/hanami/assets/sass/engine.rb

```diff
@@ -0,0 +1,37 @@
+module Hanami
+  module Assets
+    module Sass
+      # Wrapper class for Sass[C]::Engine meant to aid in the transition between
+      # Sass (EOL) and SassC as the Sass/SCSS Compiler/stylesheet compressor
+      class Engine
+        def initialize(template, options = {})
+          @renderer = source::Engine.new(template, options)
+        end
+
+        def render
+          renderer.render
+        end
+
+        def dependencies
```

Comment:
@landongrindheim This methods raises `SassC::NotRenderedError` if a `.sass` file doesn't have dependencies.

Steps to reproduce:

```shell
$ hanami new bookshelf
$ cd bookshelf
$ vim Gemfile # add sassc gem
$ vim apps/web/assets/stylesheets/foo.sass # add a simple SASS rule
$ bundle
$ bundle exec hanami assets precompile
```

We may want to catch that exception and return an empty array.

Reply:
Thanks for pointing that out! I was surprised to see that behavior, but was able to confirm it locally as well. Fixed in c340c57.

Reply:
TYVM! 👍